vmdk_plugin/E2E_Tests/e2e_verifyVolmProperties_test.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is an end-to-end test. Test will ssh to a vm and create a volume using docker cli.
+// After creating a volume, test will ssh to the ESX and verify the properties of the volume.
+// Properties being verified - capacity, disk-format and vm-attached field.
+
+package e2e_test
+
+import (
+	"fmt"
+	"github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils"
+	TestUtil "github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/test_util"
+	"log"
+	"os"
+	"testing"
+	"time"
+)
+
+var esx string = ""
+var volSizes = []string{"100MB"}
+var vms = []string{os.Getenv("VM2")}
+var formatTypes = []string{"thin", "zeroedthick", "eagerzeroedthick"}
+var vmIp string = ""
+var expctdSizeDkerCli = "100MB"
+var expctdSizeAdminCli = "100.00MB"
+var dockerVolmRmvCmd = "docker volume rm "
+
+func TestMain(m *testing.M) {
+	retCode := m.Run()
+	os.Exit(retCode)
+}
+
+// clean-up function
+func teardownFunction() {
+	fmt.Println("****clean-up**** on vm - ", vmIp)
+	for i := 0; i < len(vms); i++ {
+		TestUtil.ExecCmd(vms[i], "docker stop $(docker ps -a -q) ; docker rm $(docker ps -a -q) ; "+dockerVolmRmvCmd)
```

Comment:
is it really needed - to stop all containers? I'd expect only those we started in this test need to be stopped. 
I am not sure if there is in impact on our test pass, it is just a general clean-up expectation...